Libraries/Components/View/ViewStylePropTypes.js

```diff
@@ -34,7 +34,15 @@ var ViewStylePropTypes = {
   ),
   shadowOpacity: ReactPropTypes.number,
   shadowRadius: ReactPropTypes.number,
-  transform: ReactPropTypes.arrayOf(ReactPropTypes.object),
+  transform: ReactPropTypes.arrayOf(
+    ReactPropTypes.oneOfType([
```

Comment:
@frantic - nope, but that won't work - you need to put each transform in a separate object. Also, I have realized that `rotation` is not a prop on iOS, it looks like that was just for Android. On iOS, the `rotate` transform property exists and accepts a string: `{rotate: "1deg"}` or `{rotate: "1rad"}` just like the CSS property.

So this is how that would be written:

```javascript
<View style={{transform: [{rotate: "1deg"}, {scaleX: 2}]}} />
```

If you were to writer this, rotate would be applied by scaleX would not:

```javascript
<View style={{transform: [{rotate: "1deg", scaleX: 2}]}} />
```